client/my-sites/checkout/composite-checkout/hooks/product-variants.js

```diff
@@ -34,8 +33,7 @@ export function useProductVariants( { siteId, productSlug } ) {
 	const variantProductSlugs = useVariantPlanProductSlugs( productSlug );
 
 	const currentPlan = useSelector( ( state ) => getCurrentPlan( state, siteId ) );
-	const shouldOverride =
-		useSelector( isTreatmentInMonthlyPricingTest ) && isWpComFreePlan( currentPlan?.productSlug );
+	const shouldOverride = isWpComFreePlan( currentPlan?.productSlug );
```

Comment:
I haven't followed all the logic here, but I wonder if this variable (and its associated logic below) is still necessary? I see comments in `overridePriceForMonthlyPricingTest` that imply that maybe this is only for the test. If this line is correct, maybe those parts of the code should be changed too?

Reply:
@sirbrillig  You're right. That part should be changed. I add `replaceFullPriceWithMonthlyCost` instead of `overridePriceForMonthlyPricingTest` to update `priceFullBeforeDiscount` of yearly/biyearly dotcom plans so the full prices match in both places.
<img width="588" alt="Checkout_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/104143398-974ce980-5402-11eb-911f-0a259df6f5e2.png">

Thanks for pointing it out! 😃